### DISCUSSION TITLE
Section to adjust Manual bed leveling offset

### DISCUSSION THREAD

**Discussion by RUtarah** [Jyers/Marlin#146] [Q&A] [answered]
I already compiled my own firmware all working good but for manual mesh leveling it has different offset for every corner. Which sections that contain this offset ya?

**Comment by Jyers** [comment, marked as answer]
You can manually adjust these offset using the probing margin settings in configuration.h:
  `//#define PROBING_MARGIN_LEFT PROBING_MARGIN`
  `//#define PROBING_MARGIN_RIGHT PROBING_MARGIN`
  `//#define PROBING_MARGIN_FRONT PROBING_MARGIN`
  `//#define PROBING_MARGIN_BACK PROBING_MARGIN`

**Reply by RUtarah** [reply]
thanks sir! your firmware are awesome!
